components/dashboard/src/data/configurations/delete-configuration-mutation.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { useMutation } from "@tanstack/react-query";
+import { configurationClient } from "../../service/public-api";
+
+const BASE_KEY = "configurations";
```

Comment:
We should probably import this from configuration-queries or just put all the config related queries and mutations in that one file.